package.json

```diff
@@ -1,187 +1,176 @@
-        "@types/jasmine": "5.1.3",
-        "@types/jest": "29.4.4",
-        "@types/lodash-es": "4.17.6",
-        "@types/marked": "5.0.2",
-        "@types/node": "18.18.2",
-        "@typescript-eslint/eslint-plugin": "6.12.0",
-        "@typescript-eslint/parser": "6.12.0",
-        "@wdio/allure-reporter": "7.7.3",
-        "@wdio/cli": "7.16.16",
-        "@wdio/jasmine-framework": "7.0.0",
-        "@wdio/local-runner": "7.16.15",
-        "@wdio/sauce-service": "7.16.16",
-        "@wdio/spec-reporter": "7.7.3",
-        "ajv-formats": "2.1.1",
-        "autoprefixer": "10.4.16",
-        "axe-core": "4.5.2",
-        "axe-html-reporter": "2.2.3",
-        "axios": "1.2.0",
-        "babel-loader": "9.1.0",
-        "chromedriver": "99.0.0",
-        "conventional-changelog": "4.0.0",
-        "conventional-changelog-angular": "6.0.0",
-        "conventional-changelog-cli": "3.0.0",
-        "conventional-recommended-bump": "7.0.1",
-        "coveralls": "3.1.1",
-        "dotenv": "16.0.3",
-        "eslint": "8.46.0",
-        "eslint-config-prettier": "9.0.0",
-        "eslint-plugin-import": "2.26.0",
-        "eslint-plugin-jsdoc": "46.9.0",
-        "eslint-plugin-json": "3.1.0",
-        "eslint-plugin-markdown": "3.0.0",
-        "eslint-plugin-prefer-arrow": "1.2.3",
-        "fast-glob": "3.3.1",
-        "fs-extra": "11.1.1",
-        "git-semver-tags": "7.0.1",
-        "github-assistant": "0.3.4",
-        "husky": "8.0.2",
-        "jasmine-core": "4.5.0",
-        "jasmine-marbles": "0.9.2",
-        "jasmine-spec-reporter": "7.0.0",
-        "jest": "29.7.0",
-        "jest-environment-jsdom": "29.7.0",
-        "jest-environment-node": "^29.4.1",
-        "jest-extended": "4.0.2",
-        "jest-preset-angular": "13.1.4",
-        "lerna": "7.4.2",
-        "lint-staged": "15.1.0",
-        "nan": "2.18.0",
-        "ng-packagr": "17.0.2",
-        "ngx-cva-test-suite": "2.0.0",
-        "nx": "17.1.3",
-        "postcss": "^8.4.21",
-        "postcss-import": "15.1.0",
-        "postcss-preset-env": "9.0.0",
-        "postcss-url": "10.1.3",
-        "prettier": "3.1.0",
-        "prettier-plugin-organize-imports": "3.2.4",
-        "rollup": "3.26.3",
-        "semver": "7.5.4",
-        "standard-version": "9.5.0",
-        "through2": "4.0.2",
-        "ts-jest": "29.1.0",
-        "ts-node": "10.9.1",
-        "tsconfig-paths": "4.1.0",
-        "typedoc": "0.25.3",
-        "typedoc-plugin-merge-modules": "5.1.0",
-        "typescript": "5.2.2",
-        "wait-on": "6.0.1",
-        "wdio-axe": "2.2.3",
-        "wdio-chromedriver-service": "6.0.4",
-        "webdriverio": "7.27.0"
-    },
-    "lint-staged": {
-        "*": [
-            "yarn nx format:write --uncommitted"
-        ],
-        "*.{js,ts,html,md,json,yml}": [
-            "yarn eslint --fix"
-        ],
-        "*.properties": [
-            "yarn nx run i18n:transform-translations"
-        ]
-    }
+  "name": "fundamental-ngx",
```

Comment:
we should avoid reformatting the file all the time

Reply:
If it's gonna be 2 spaces all the time, no other changes should be made. This is done by our formatters